akka-cluster-bootstrap/src/main/scala/com/lightbend/rp/akkaclusterbootstrap/ClusterStatusCheck.scala

```diff
@@ -24,24 +24,24 @@ import scala.concurrent.{ ExecutionContext, Future }
 class ClusterStatusCheck extends ReadinessCheck with HealthCheck {
   def healthy(actorSystem: ExtendedActorSystem)(implicit ec: ExecutionContext): Future[Boolean] =
     status(actorSystem) map {
-      case MemberStatus.Joining  => true
+      case MemberStatus.Joining => true
       case MemberStatus.WeaklyUp => true
-      case MemberStatus.Up       => true
-      case MemberStatus.Leaving  => true
-      case MemberStatus.Exiting  => true
-      case MemberStatus.Down     => false
-      case MemberStatus.Removed  => false
+      case MemberStatus.Up => true
+      case MemberStatus.Leaving => true
+      case MemberStatus.Exiting => true
+      case MemberStatus.Down => false
+      case MemberStatus.Removed => false
     }
 
   def ready(actorSystem: ExtendedActorSystem)(implicit ec: ExecutionContext): Future[Boolean] =
     status(actorSystem) map {
-      case MemberStatus.Joining  => false
+      case MemberStatus.Joining => false
       case MemberStatus.WeaklyUp => true
-      case MemberStatus.Up       => true
-      case MemberStatus.Leaving  => false
-      case MemberStatus.Exiting  => false
-      case MemberStatus.Down     => false
-      case MemberStatus.Removed  => false
+      case MemberStatus.Up => true
+      case MemberStatus.Leaving => false
+      case MemberStatus.Exiting => false
+      case MemberStatus.Down => false
+      case MemberStatus.Removed => false
     }
```

Comment:
try not to break formatting please, makes it harder to track changes

Reply:
I am committing what sbt-scalariform did. Not sure how @dwijnand got this to compile in 72e67d236bf6df49851f47ef7352002335ce7439.

Reply:
Formatting isn't enforced by CI.